src/data/faq_de.json

```
@@ -50,11 +120,70 @@
+                                    "Sie können andere CWA-Nutzende noch bis einschließlich 30. April 2023 über ein positives Testergebnis informieren und diese damit warnen."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für die Corona-Warn-App noch Event-QR-Codes ausstellen?",
+                                "anchor": "ramp_down_qr_codes",
+                                "active": false,
+                                "textblock": [
+                                    "Eine Generierung neuer QR-Codes wird innerhalb der App bis einschließlich 30. April 2023 möglich sein."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich bei einem Event-QR-Code noch einchecken und Andere warnen?",
+                                "anchor": "ramp_down_check_in",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, ein Warnen über die Event-QR-Codes ist aufgrund der fehlenden Infrastruktur nicht mehr möglich. Daher wird die App diese Funktion im Release 3.2 ab 1. Mai 2023 auch nicht mehr anbieten."
+                                ]
+                            },
+                            {
+                                "title": "Was ist, wenn die App Störungen/Fehler hat; werden diese dann nicht mehr beseitigt? Kann ich diese weiterhin melden?",
+                                "anchor": "ramp_down_reports",
+                                "active": false,
+                                "textblock": [
+                                    "Bis einschließlich 30. April.2023 können aus der App heraus noch Fehlerberichte an die Entwicklung übertragen werden. Danach ist es zwar möglich, eventuelle Fehler und Störungen bis einschließlich 31. Mai 2023 zu melden. Jedoch können Bugfixes zur noch für solche Fehler erstellt werden, die bis einschließlich 15. Mai 2023 gemeldet wurden. Entsprechend ist es auch in GitHub ab 16. Mai 2023 nicht mehr möglich, neue Issues zu eröffnen."
+                                ]
+                            },
+                            {
+                                "title": "Werden dann auch keine Zufalls-Kennungen für Begegnungen mehr ausgetauscht, wenn ich die App lösche?",
+                                "anchor": "ramp_down_exchange",
+                                "active": false,
+                                "textblock": [
+                                    "Der Austausch der Zufalls-Kennungen zur sog. Kontaktnachverfolgung (engl. Contact Tracing) erfolgt technisch durch das ENF-Modul (Exposure Notification Framework) Ihres Betriebssystems. Um sicherzustellen, dass nach einer Deinstallation der Corona-Warn-App auch das ENF-Modul deaktiviert ist, gehen Sie bitte wie folgt vor:",

```

"ENF-Modul" is a completely new word introduced only in this FAQ entry, I suggest just using the full name "Exposure Notification Framework".

---

"ENF" is used in the FAQs in other places

---

ENF is totally fine.

---

Perfect, thanks for confirming @GisoSchroederSAP! 

Suggestion: 

```suggestion
                                    "Der Austausch der Zufalls-Kennungen zur sog. Kontaktnachverfolgung (engl. Contact Tracing) erfolgt technisch durch das ENF (Exposure Notification Framework) Ihres Betriebssystems. Um sicherzustellen, dass nach einer Deinstallation der Corona-Warn-App auch das ENF-Modul deaktiviert ist, gehen Sie bitte wie folgt vor:",
```

---

@Ein-Tim 
There is still one instance of `ENF-Modul` left in your text.

---

I'm pretty sure, you guys will find them all and will clean them up, thank you!

---

@GisoSchroederSAP 

> I'm pretty sure, you guys will find them all and will clean them up, thank you!

It seems that was a challenge to use an editor search, and you were right that the next line also contains this term.

---

Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/73c82ad32454368aa9de62973567f834a1c8e0ca, thanks!